README.md

```diff
@@ -1,34 +1,19 @@
 # node-minecraft-data
+
 [![NPM version](https://badge.fury.io/js/minecraft-data.svg)](http://badge.fury.io/js/minecraft-data)
 [![Tonic](https://img.shields.io/badge/tonic-try%20it-blue.svg)](https://tonicdev.com/npm/minecraft-data)
 [![Build Status](https://github.com/PrismarineJS/node-minecraft-data/workflows/CI/badge.svg)](https://github.com/PrismarineJS/node-minecraft-data/actions?query=workflow%3A%22CI%22)
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/node-minecraft-data)
 
-node-minecraft-data provides easy access to [minecraft-data](https://github.com/PrismarineJS/minecraft-data) in node.js.
+`node-minecraft-data` provides easy access to [minecraft-data](https://github.com/PrismarineJS/minecraft-data) in node.js.
 
-The objective of this module is to make easier to look for information in minecraft-data in node.
+The objective of this module is to make easier to look for information in `minecraft-data` in node.
 
 ## Features
 
-For example it's often useful to :
-
-* find blocks by id
-* find items by name
-* find block or item by name
-* find block or item by id
-
-## Example
```

Comment:
Why did you remove that?

Reply:
Just thought it was unnecessary, as it is mentioned that below. Are you specifically referring to the example or the possible uses list?

Reply:
Specifically referring to the examples. They're here to help people start faster